jekyll/_cci2/migrating-from-github.adoc

```diff
@@ -0,0 +1,288 @@
+
+We have various other features that set our solution apart. https://circleci.com/signup/[Sign up for a free account today] and try us out, or if you are interested in CircleCI for your team, https://circleci.com/talk-to-us/?source-button=MigratingFromGithubActionsDoc[contact our sales team] to set up a trial.
+
+== Concepts
+
+=== Jobs and Workflows
+
+Both Github Actions and CircleCI share similar concepts around "jobs" and "workflows". A workflow is an end-to-end flow of connected jobs, which in turn consist of commands to achieve an atomic task (e.g. "run unit tests" or "build a Docker image").
+
+CircleCI differs primarily in configuration syntax, setting up workflow and job dependencies in a separate section as opposed to inline in the job.
+
+[.table.table-striped]
+[cols=2*, options="header", stripes=even]
+[cols="50%,50%"]
+|===
+| Github | CircleCI
+
+a|
+[source, yaml]
+----
+name: My GitHub Actions Workflow
+
+on: [push]
+
+jobs:
+  job_1:
+    runs-on: ubuntu-latest
+    steps:
+      # job steps
+  job_2:
+    needs: job_1
+    runs-on: ubuntu-latest 
+    steps:
+      # job steps
+----
+
+a|
+[source, yaml]
+----
+jobs:
+  job_1:
+    executor: my-ubuntu-exec
+    steps:
+      # job steps
+  job_2:
+    executor: my-ubuntu-exec
+    steps:
+      # job steps
+
+workflows:
+  my_workflow:
+    jobs:
+      - job_1
+      - job_2:
+          requires:
+            - job_1
+----
+|===
+
+=== Actions vs. Orbs
+"Actions" in Github are reusable commands or tasks to run inside a job. However, they are written for execution inside a Docker container or coded as individual steps using JavaScript. This adds additional work and limits the scope in which they can be applied.
+
+CircleCI offers similar functionality in our https://circleci.com/docs/2.0/orb-intro/#section=configuration[Orbs]. The primary difference is that CircleCI Orbs are just packaged, reusable YAML, so you can orbify reusable jobs, executors, or commands, and use them however you see fit in any of your jobs or workflows.
+
+Github offers browsing of Actions in their Marketplace; CircleCI has an https://circleci.com/orbs/registry/[Orb Registry] as well as an https://circleci.com/integrations/[Integrations Page] containing numerous Certified, Partner, and community Orbs / Integrations.
+
+=== Runners vs. Executors
+In GitHub, you can specify your builds to run in Linux, macOS, and Windows environments via a `runs-on` key in the YAML, and if you want to run anything in a container, you specify an additional `container` key.
+
+In CircleCI, you have the same choice of environments (called Executors), except that Docker containers are their own class of executor.
```

Comment:
`except that Docker containers are their own class of executor.` -> `with additional options and features for Docker`